accounts/tests.py

```diff
@@ -1,3 +1,43 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User # noqa
+from .models import UserAccount
+from . import signals # noqa
 
-# Create your tests here.
+
+@pytest.fixture
+def test_account(db):
+    username = 'testuser'
+    password = 'lvkmdgv32fdbf1b'
+    email = 'testuser@test.com'
+    new_user = User.objects.create_user(username=username,
+                                        password=password,
+                                        email=email)
+    return new_user
+
+
+@pytest.mark.django_db()
+def test_account_create(test_account):
+    test_account
```

Comment:
What's the reason for this `test_account`? (same for all other tests)

Reply:
Originally it was supposed to be for less redundancy(creating a test user with the same details every time) but I'll remove it and try something else

Reply:
It's already done by calling the fixture in the parentheses
`def test_account_create(test_account):`